content/en/continuous_integration/intelligent_test_runner/javascript.md

```diff
@@ -15,17 +15,17 @@ further_reading:
 Intelligent Test Runner is only supported in the following versions and testing frameworks:
 
 * `jest>=24.8.0`
-  * From `dd-trace>=3.16.0` or `dd-trace>=2.29.0`.
+  * From `dd-trace>=4.16.0` or `dd-trace>=3.37.0`.
```

Comment:
This is very weird to me.

ITR support's available much earlier than this. We're suddenly bumping up the minimum version requirement just so we can remove the app key from the docs?

I guess we can say "these docs only apply from version X and later" but mayve we should keep the older docs around and say that the requirement for the app key was removed in version X.

Reply:
If we're going to do things this way, then maybe what we should actually be doing is documenting the tracer configuration in each tracer's versioned docs (eg: https://ddtrace.readthedocs.io/en/v1.18.0/integrations.html#pytest for `dd-trace-py==1.18.0`).

Reply:
this is something that bugs me too and we have no good solution for it. If we wanted to do it properly we'd have to have a huge matrix of versions and features. 

I think that the doc site is set up to promote people to bump to the highest possible version without breaking their setup. I think yours is a fair point though

Reply:
I guess it's not a big deal because it's not a breaking change (in the sense that someone who still has an app key won't find themselves in a broken state). The only risk is someone with an older version if a tracer trying out ITR and running into a missing app key error, at which point I think the discoverability of what's actually going on isn't great, but maybe that's not something to really worry about.